metadata.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 Canonical
+# See LICENSE file for licensing details.
+
+name: tls-certificates-interface
+description: |
+  Placeholder charm
```

Comment:
Isn't this a 
```suggestion
  Tester charm for tls-certificates-interface
```
?

Reply:
I mean the charm itself is a placeholder, no-one would actually deploy it. It is true though that it can be used for testing but the typical usecase would be people fetching its libs.

Reply:
@sed-i Can you please check my answers to your comment and resolve/approve the PR if you don't have more?